site/content/en/contributions/design/backend.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Backend "
+---
+
+## Overview
+
+This design document introduces the `Backend` API allowing system administrators to represent backends without the use 
+of a K8s `Service` resource. 
+
+Common use cases for non-Service backends in the K8s and Envoy ecosystem include:
+- Cluster-external endpoints, which are currently second-class citizens in Gateway-API 
+  (supported using [Services and FQDN endpoints][]).
+- Host-local endpoints, such as sidecars or daemons that listen on [unix domain sockets][] or envoy [internal listeners][], 
+  that cannot be represented by a K8s service at all.
+
+Several projects currently support backends that are not registered in the infrastructure-specific service registry. 
+- K8s Ingress: [Resource Backends][]
+- Istio: [Service Entry][]
+- Gloo Edge: [Upstream][]
+- Consul: [External Services][]
+
+## Goals
+* Add an API definition to hold settings for configuring Unix Domain Socket, FQDN and IP.
+* Determine which resources may reference the new backend resource.
+* Determine which existing Gateway-API and Envoy Gateway policies may attach to the new backend resource. 
+
+## Non Goals
+* Support specific backend types, such as S3 Bucket, Redis, AMQP, InfluxDB, etc.  
+
+## Implementation
+
+The `Backend` resource is an implementation-specific Gateway-API [BackendObjectReference Extension][]. 
+
+### Example
+Here is an example highlighting how a user can configure a route that forwards traffic to both a Service and a local
```

Comment:
curious about here: do you mean we can configure a route that forwards traffic to a Service `backend-svc`, a locl UDS, and `10.244.0.28:3000`?

Reply:
yes, thanks, fixed